packages/ifpack2/src/Ifpack2_AdditiveSchwarz_def.hpp

```diff
@@ -966,6 +968,9 @@ setParameterList (const Teuchos::RCP<Teuchos::ParameterList>& plist)
   // singletons should help for PDE problems with Dirichlet BCs.
   FilterSingletons_ = plist->get ("schwarz: filter singletons", FilterSingletons_);
 
+  // Allow for damped Schwarz updates
+  UpdateDamping_ = Teuchos::as<scalar_type>(plist->get("schwarz: update damping",UpdateDamping_));
```

Comment:
Would it ever make sense for the damping parameter to have a nonzero imaginary part?  If not, then it would be nicer if we made this `magnitude_type` with a fall-back to `double`.